dashboard/test/models/level_concept_difficulty_test.rb

```diff
@@ -17,6 +17,40 @@ class LevelConceptDifficultyTest < ActiveSupport::TestCase
     assert_equal @level_concept_difficulty.sequencing, 1
   end
 
+  test 'gets attributes when created as part of Level' do
+    # Newly-created level has no LevelConceptDifficulty by default.
+    level = create :level
+    assert_nil level.level_concept_difficulty
+
+    # Assigning a level_concept_difficulty hash to a Level without one
+    # creates a new LevelConceptDifficulty with appropriate properties.
+    level.assign_attributes(
+      'level_concept_difficulty' => {'sequencing' => 3}
+    )
+    refute_nil level.level_concept_difficulty
+    assert_equal 3, level.level_concept_difficulty.sequencing
```

Comment:
Out of these two tests, this is the only assertion that was failing prior to my fix.